src/api/operator/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as realApi from './operatorApi'
+import * as mockApi from './operatorMock'
+export * from './types'
+
+const useMock = process.env.MOCK_OPERATOR
+
+// Re-exporting the result, mocked or not.
+// Unfortunately, did not find a way to export
+// a mix of mock/real implementations
+// without manually naming the exports
+export const {
+  // functions that have a mock
+  getOrder,
+  getOrders,
+  // functions that do not have a mock
+  getOrderLink = realApi.getOrderLink,
+  postSignedOrder = realApi.postSignedOrder,
+  getFeeQuote = realApi.getFeeQuote,
+  // function that only have a mock
+  getTrades = mockApi.getTrades,
+} = useMock ? { ...mockApi } : { ...realApi }
```

Comment:
Aha, ok, u want to maintain the useMock thing. 

My initial idea was more simple, just mock the services that are not yet implemented. But happy to do it this way too if we think is useful

Reply:
Yeah, we might need to mock stuff that's not yet available or hard to reproduce, such as the `getOrder` calls that disappear very quickly.


Not very happy with this solution though, as every new function will need to be manually added to this list of exports.
Guess that's a down side of the functional style api?

Reply:
why spread into object instead of returning the object directly?

Reply:
Because of Typescript.
Since not all implemented APIS are mocked and not all mocked APIs are implemented, there's a type mismatch.
I sold this by providing a default for either case, but TS doesn't like that.